packages/core/core-flows/src/product/workflows/create-collections.ts

```diff
@@ -2,18 +2,28 @@ import { ProductTypes } from "@medusajs/types"
 import {
   WorkflowData,
   WorkflowResponse,
+  createHook,
   createWorkflow,
 } from "@medusajs/workflows-sdk"
 import { createCollectionsStep } from "../steps"
 
-type WorkflowInput = { collections: ProductTypes.CreateProductCollectionDTO[] }
+type WorkflowInput = {
+  collections: ProductTypes.CreateProductCollectionDTO[]
+  additional_data?: Record<string, unknown>
+}
 
 export const createCollectionsWorkflowId = "create-collections"
 export const createCollectionsWorkflow = createWorkflow(
   createCollectionsWorkflowId,
-  (
-    input: WorkflowData<WorkflowInput>
-  ): WorkflowResponse<ProductTypes.ProductCollectionDTO[]> => {
-    return new WorkflowResponse(createCollectionsStep(input.collections))
+  (input: WorkflowData<WorkflowInput>) => {
+    const collections = createCollectionsStep(input.collections)
+    const collectionsStepCreated = createHook("collectionsStepCreated", {
```

Comment:
suggestion: in some cases the hook name include `step` in other no, should we unify?

Reply:
would suggest that we don't use the notion of a step in the hook names

Reply:
> would suggest that we don't use the notion of a step in the hook names

+1

Reply:
> suggestion: in some cases the hook name include step in other no, should we unify?

Was a typo :)